src/hotspot/cpu/aarch64/interp_masm_aarch64.cpp

```diff
@@ -692,6 +692,12 @@ void InterpreterMacroAssembler::remove_activation(
     // testing if reserved zone needs to be re-enabled
     Label no_reserved_zone_enabling;
 
+    // check if already enabled - if so no re-enabling needed
```

Comment:
```suggestion
    // check if already enabled - if so, no re-enabling is needed
```

Reply:
Yeah, but this is a clean backport, so I would not deviate :)